awx/ui/src/screens/Template/WorkflowJobTemplateDetail/WorkflowJobTemplateDetail.js

```diff
@@ -202,26 +201,25 @@ function WorkflowJobTemplateDetail({ template }) {
             helpText={helpText.enabledOptions}
           />
         )}
-        {summary_fields.labels?.results?.length > 0 && (
-          <Detail
-            fullWidth
-            label={t`Labels`}
-            helpText={helpText.labels}
-            value={
-              <ChipGroup
-                numChips={3}
-                totalChips={summary_fields.labels.results.length}
-                ouiaId="workflow-job-template-detail-label-chips"
-              >
-                {summary_fields.labels.results.map((l) => (
-                  <Chip key={l.id} ouiaId={`${l.name}-label-chip`} isReadOnly>
-                    {l.name}
-                  </Chip>
-                ))}
-              </ChipGroup>
-            }
-          />
-        )}
+        <Detail
+          fullWidth
+          label={t`Labels`}
+          helpText={helpText.labels}
+          value={
+            <ChipGroup
+              numChips={3}
+              totalChips={summary_fields.labels.results.length}
+              ouiaId="workflow-job-template-detail-label-chips"
+            >
+              {summary_fields.labels.results.map((l) => (
+                <Chip key={l.id} ouiaId={`${l.name}-label-chip`} isReadOnly>
+                  {l.name}
+                </Chip>
+              ))}
+            </ChipGroup>
+          }
+          isEmpty={summary_fields.labels?.results?.length === 0}
```

Comment:
if `summary_fields.labels` is null/undefined, this won't work correctly since `undefined === 0` is false. It's probably better to do `isEmpty={!summary_fields.labels?.results?.length}`